substrate/pkg/utils/helm/chart.go

```diff
@@ -89,3 +89,11 @@ func (c *Client) Apply(ctx context.Context, chart *Chart) error {
 	logging.FromContext(ctx).Infof("Upgraded chart %s/%s", chart.Repository, chart.Name)
 	return nil
 }
+
+func Affinity() map[string]interface{} {
```

Comment:
Can we use node selector instead off affinity as its more cleaner and provide the management node label.

Reply:
Good point. I think the node selector and the node anti-affinity are both needed.
Having the node selector make sure that kit operator always lands on the control plane node.
Having the anti-affinity make sure that we don't have an implicit restricted label `"kit.aws/substrate": "control-plane"`